lib/core/mqtt_topics.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+namespace hyped::core {
+
+enum class MqttTopic { kTest };
+
+static std::unordered_map<MqttTopic, std::string> mqtt_topic_to_string
```

Comment:
`const static` for both of these, maybe `constexpr` if it compiles but i don't think it does

Reply:
this is the same issue that cropped up [here](https://github.com/Hyp-ed/hyped-2024/pull/23#discussion_r1388694526), do we want them static or const? as having them as both complicates things (define and instantiate in different files)

Reply:
`const` then.
Reading into it, I think a `const` variable at the namespace level is implicitly `static` cos it's obviously only instantiated once. 

(explicitly using the `static` keyword in namespace scope limits it's visibility to the current translation unit only, but what that actually impacts semantically i'm not sure)